README.md

```diff
@@ -24,6 +24,23 @@ The package could be installed with composer:
 composer require yiisoft/translator-formatter-simple
 ```
 
+## Configuration
+
+Example of config, if you not use package [`yiisoft/config`](http://github.com/yiisoft/config)
```

Comment:
```suggestion
In case you use [`yiisoft/config`](http://github.com/yiisoft/config), you will get configuration automatically. If not, the following DI container configuration is necessary:
```